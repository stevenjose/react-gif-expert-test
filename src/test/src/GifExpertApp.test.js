import React from "react";
import { shallow } from "enzyme/build";
import { GifExpertApp } from "../../GifExpertApp";
import "@testing-library/jest-dom";
import { AddCategory } from "../../components/AddCategory";

jest.mock("../../components/AddCategory");
const wrapper = shallow(<GifExpertApp />);

describe("pruebas en el componente gitexpert", () => {
  test("pruebas con el componente", () => {
    expect(wrapper).toMatchSnapshot();
  });

  test("exista el componente AddCategory", () => {
    expect(wrapper.find('AddCategory').exists()).toBe(true);
  });

  test('probar componente GifGrid', () => {
    expect(wrapper.find('GifGrid').length).toBe(1);
  });
});
