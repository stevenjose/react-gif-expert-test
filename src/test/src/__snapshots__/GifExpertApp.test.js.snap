// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en el componente gitexpert pruebas con el componente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="One Punch"
      key="One Punch"
    />
  </ol>
</Fragment>
`;
